cmake/LLVMIRUtil.cmake

```diff
@@ -1007,6 +1034,6 @@ function(llvmir_attach_library)
     set_property(SOURCE ${IN_FULL_LLVMIR_FILE} PROPERTY EXTERNAL_OBJECT TRUE)
   endforeach()
 
-  ## postamble
+# # postamble
```

Comment:
something weird with your formatting here

Reply:
Fixed in the new commit. Sorry for my auto-formatting tool... not sure what's wrong with it